pxe-server/templates/configmap.yaml

```diff
@@ -0,0 +1,30 @@
+# SPDX-FileCopyrightText: (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: pxe-server-config
+data:
+  dnsmasq.conf: |
+    interface={{ required "config.interface is mandatory!" .Values.config.interface }}
+    bind-interfaces
+    port=0
+    # subnetAddress is a network address of an IP subnet that is used to match
+    # incoming DHCP requests. Only DHCP requests coming from hosts in the subnet will be handled.
+    # The line below is used to turn dnsmasq into ProxyDHCP mode. 
+    dhcp-range={{ required "config.subnetAddress is mandatory!" .Values.config.subnetAddress }},proxy
+    
+    leasefile-ro
+    no-hosts
+    log-dhcp
+    log-queries
+    log-facility=-
+    log-debug
+    enable-tftp
+    tftp-root=/var/lib/tftp
```

Comment:
are we implementing resource limiter (CPU and memory) for the boots and pxe server to mitigate from attacks ?

Reply:
Good point, adding now